docs/source/requirements-single.rst

```diff
@@ -20,7 +20,7 @@ The controller is where the Ansible engine that runs the playbook is installed.
 Refer to RedHat Ansible Certified Content documentation for more on the `controllers dependencies`_.
 
 .. _controllers dependencies:
-   https://ibm.github.io/z_ansible_collections_doc/requirements/requirements_controller.html
+   https://ibm.github.io/z_ansible_collections_doc/requirements/requirements.html#control-node
 .. ...........................................................................
 .. © Copyright IBM Corporation 2020                                          .
```

Comment:
shouldn't this be 2020-2024 on copyright?

Reply:
Good point, looks like sphynx only supports one year, so the latest, format they allow is via the config:
```
A copyright statement in the style '2008, Author Name'.

Changed in version 7.1: The value may now be a sequence of copyright statements in the above form, which will be displayed each to their own line.
```

Ill update it to:
```
copyright = '2024, IBM'
```
See doc [link](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-copyright).